lib/utils.js

```diff
@@ -246,7 +246,11 @@ Changes.prototype.notify = function (dbName) {
   this.notifyLocalWindows(dbName);
 };
 
-if (typeof window === 'undefined' || typeof window.atob !== 'function') {
+if (typeof atob === 'function' || typeof window !== 'undefined') {
+  exports.atob = function (str) {
+    return atob(str);
+  };
```

Comment:
also this can be simplified to `exports.atob = atob;` feel dumb for missing that :P